pkg/cmd/pipeline/describe_test.go

```diff
@@ -321,6 +321,8 @@ func TestPipelineDescribe_with_multiple_v1alpha1_pipelineruns(t *testing.T) {
 				tb.PipelineDescription("a test description"),
 				tb.PipelineTask("task", "taskref",
 					tb.RunAfter("one", "two"),
+					tb.PipelineTaskTimeout(5*time.Minute),
```

Comment:
Could you also please include a `PipelineTaskTimeout` in a `v1beta1` describe test?